2024-12-28T19:27:52.658326-0300 INFO Chamando função 'soma' com args (1, 4) e kwargs{} main_logs.py
2024-12-28T19:27:52.659206-0300 INFO Função 'soma retornou 5 main_logs.py
2024-12-28T19:27:52.659715-0300 INFO Chamando função 'soma' com args (6, '9') e kwargs{} main_logs.py
2024-12-28T19:27:52.660291-0300 ERROR Exceção capturada em 'soma': unsupported operand type(s) for +: 'int' and 'str' main_logs.py
2024-12-28T19:28:13.565728-0300 INFO Chamando função 'soma' com args (1, 4) e kwargs{} main_logs.py
2024-12-28T19:28:13.566620-0300 INFO Função 'soma retornou 5 main_logs.py
2024-12-28T19:28:13.567301-0300 INFO Chamando função 'soma' com args (6, '9') e kwargs{} main_logs.py
2024-12-28T19:28:13.568031-0300 ERROR Exceção capturada em 'soma': unsupported operand type(s) for +: 'int' and 'str' main_logs.py
Traceback (most recent call last):

  File "D:\bootcamp_python\aula_09\exemplo_00.py", line 8, in <module>
    soma(6,"9")
    └ <function soma at 0x000001FF3EAE93A0>

> File "D:\bootcamp_python\aula_09\main_logs.py", line 27, in wrapper
    result = func(*args,**kwargs)
             │     │      └ {}
             │     └ (6, '9')
             └ <function soma at 0x000001FF3EAE39C0>

  File "D:\bootcamp_python\aula_09\exemplo_00.py", line 5, in soma
    return x + y
           │   └ '9'
           └ 6

TypeError: unsupported operand type(s) for +: 'int' and 'str'
2024-12-28T19:40:20.172121-0300 INFO Chamando função 'soma' com args (1, 4) e kwargs{} main_logs.py
2024-12-28T19:40:20.172867-0300 INFO Função soma executada em -1.1920928955078125e-06 main_logs.py
2024-12-28T19:40:20.173392-0300 INFO Função 'soma retornou 5 main_logs.py
2024-12-28T19:40:20.173921-0300 INFO Chamando função 'soma' com args (6, '9') e kwargs{} main_logs.py
2024-12-28T19:40:20.174543-0300 ERROR Exceção capturada em 'soma': unsupported operand type(s) for +: 'int' and 'str' main_logs.py
Traceback (most recent call last):

  File "D:\bootcamp_python\aula_09\exemplo_00.py", line 9, in <module>
    soma(6,"9")
    └ <function soma at 0x000002BCA69794E0>

> File "D:\bootcamp_python\aula_09\main_logs.py", line 27, in wrapper
    result = func(*args,**kwargs)
             │     │      └ {}
             │     └ (6, '9')
             └ <function soma at 0x000002BCA69793A0>

  File "D:\bootcamp_python\aula_09\main_logs.py", line 39, in wrapper
    result = func(*args,**kwargs)
             │     │      └ {}
             │     └ (6, '9')
             └ <function soma at 0x000002BCA69739C0>

  File "D:\bootcamp_python\aula_09\exemplo_00.py", line 6, in soma
    return x + y
           │   └ '9'
           └ 6

TypeError: unsupported operand type(s) for +: 'int' and 'str'
2024-12-28T19:41:09.109783-0300 INFO Chamando função 'soma' com args (1, 4) e kwargs{} main_logs.py
2024-12-28T19:41:09.110388-0300 INFO Função soma executada em -7.152557373046875e-07 main_logs.py
2024-12-28T19:41:09.110930-0300 INFO Função 'soma retornou 5 main_logs.py
2024-12-28T19:41:09.111490-0300 INFO Chamando função 'soma' com args (6, 9) e kwargs{} main_logs.py
2024-12-28T19:41:09.112090-0300 INFO Função soma executada em -7.152557373046875e-07 main_logs.py
2024-12-28T19:41:09.112593-0300 INFO Função 'soma retornou 15 main_logs.py
2024-12-28T19:41:09.113080-0300 INFO Chamando função 'soma' com args (322, 19293) e kwargs{} main_logs.py
2024-12-28T19:41:09.113651-0300 INFO Função soma executada em -4.76837158203125e-07 main_logs.py
2024-12-28T19:41:09.114186-0300 INFO Função 'soma retornou 19615 main_logs.py
2024-12-28T19:41:34.726296-0300 INFO Chamando função 'soma' com args (1, 4) e kwargs{} main_logs.py
2024-12-28T19:41:34.726961-0300 INFO Função soma executada em -0.0000 segundos main_logs.py
2024-12-28T19:41:34.727499-0300 INFO Função 'soma retornou 5 main_logs.py
2024-12-28T19:41:34.728020-0300 INFO Chamando função 'soma' com args (6, 9) e kwargs{} main_logs.py
2024-12-28T19:41:34.728497-0300 INFO Função soma executada em -0.0000 segundos main_logs.py
2024-12-28T19:41:34.729145-0300 INFO Função 'soma retornou 15 main_logs.py
2024-12-28T19:41:34.729655-0300 INFO Chamando função 'soma' com args (322, 19293) e kwargs{} main_logs.py
2024-12-28T19:41:34.730236-0300 INFO Função soma executada em -0.0000 segundos main_logs.py
2024-12-28T19:41:34.731365-0300 INFO Função 'soma retornou 19615 main_logs.py
